### PR TITLE
Fixes the NailGun Again

### PR DESCRIPTION
**Pull request by Lukas-Seltz**
# About the pull request

Fixes the CE Vendor So he can get the Nailgun and the Laser Designator.

# Explain why it's good for the game

Because not being able to get Something you should get is bad.

# Testing Photographs and Procedure

# Changelog
:cl:
fix: Fixed the CE vendor to be able to vend out what you should be able to. 
/:cl:

### CODE REVIEW

**Comment by fira** [comment]
Won't that
 1. Let you buy 99 of these
 2. Prevent you eventually from using Dress Vendor
?
This one is handled specially

**Comment by Lukas-Seltz** [comment]
> Won't that
> 
> 1. Let you buy 99 of these
> 2. Prevent you eventually from using Dress Vendor
>    ?
>    This one is handled specially

The problem is that with the old one, it links the Laser Designator and the Nail gun. I am working to figure out a better way to do it.

**Comment by fira** [comment]
We merged multi-vending a while back so you could just bundle it in KIT with the PDA by making it a list

**Comment by fira** [comment]
Example `list("Standard Marine Apparel", 0, list(/obj/item/clothing/under/marine, /obj/item/clothing/shoes/marine/knife, /obj/item/clothing/gloves/marine, /obj/item/device/radio/headset/almayer/marine), MARINE_CAN_BUY_UNIFORM, VENDOR_ITEM_MANDATORY),`

**Comment by Lukas-Seltz** [comment]
While true I do not see a need to bundle the PDA and nail gun together

**Comment by fira** [comment]
Well the need is just avoiding adding a MARINE_CAN_BUY_NAILGUN overspecific entry to every human in the game

**Review by harryob — CHANGES_REQUESTED** [review]
firas changes

File: code/game/machinery/vending/vendor_types/crew/senior_officers.dm [hunk 1]
```diff
@@ -85,12 +85,7 @@ GLOBAL_LIST_INIT(cm_vending_clothing_chief_engineer, list(
 		list("SHIPSIDE GEAR", 0, null, null, null),
 
 		list("STANDARD EQUIPMENT (TAKE ALL)", 0, null, null, null),
-		list("Insulated Gloves", 0, /obj/item/clothing/gloves/yellow, MARINE_CAN_BUY_GLOVES, VENDOR_ITEM_MANDATORY),
-		list("Headset", 0, /obj/item/device/radio/headset/almayer/ce, MARINE_CAN_BUY_EAR, VENDOR_ITEM_MANDATORY),
-		list("Marine Combat Boots", 0, /obj/item/clothing/shoes/marine, MARINE_CAN_BUY_SHOES, VENDOR_ITEM_MANDATORY),
-		list("Welding Goggles", 0, /obj/item/clothing/glasses/welding, MARINE_CAN_BUY_GLASSES, VENDOR_ITEM_MANDATORY),
-		list("KN5500/2 PDA", 0, /obj/item/device/working_joe_pda/uscm, MARINE_CAN_BUY_KIT, VENDOR_ITEM_MANDATORY),
-		list("Tactical Compact Nailgun", 0, /obj/item/weapon/gun/smg/nailgun/compact/tactical, MARINE_CAN_BUY_MRE, VENDOR_ITEM_MANDATORY),
+		list("Standard Equipment", 0, list(/obj/item/clothing/gloves/yellow, /obj/item/device/radio/headset/almayer/ce,  /obj/item/clothing/shoes/marine, /obj/item/clothing/glasses/welding, /obj/item/device/working_joe_pda/uscm, /obj/item/weapon/gun/smg/nailgun/compact/tactical, MARINE_CAN_BUY_SHOES, VENDOR_ITEM_MANDATORY)),
```
Comment (kiVts) [inline code/game/machinery/vending/vendor_types/crew/senior_officers.dm:88]:
```suggestion
		list("Standard Equipment", 0, list(/obj/item/clothing/gloves/yellow, /obj/item/device/radio/headset/almayer/ce,  /obj/item/clothing/shoes/marine, /obj/item/clothing/glasses/welding, /obj/item/device/working_joe_pda/uscm, /obj/item/weapon/gun/smg/nailgun/compact/tactical), MARINE_CAN_BUY_SHOES, VENDOR_ITEM_MANDATORY),
```

**Review by kiVts — CHANGES_REQUESTED** [review]

**Review by kiVts — APPROVED** [review]

**Pull request merged**